requirements.txt

```diff
@@ -1 +1,2 @@
 ops-lib-pgsql
+pydantic==1.10.14
```

Comment:
Im wondering if we should pin all the way down to patch versions - should we be using poetry for this?

Reply:
In this case we should pin down due to SAML requiring pydantic<2.